apps/books/templates/categoryvisebooks.html

```diff
@@ -0,0 +1,28 @@
+<h1>{{ object.title }}:</h1>
+<ul style="list-style-type: none;">
+    {% for book in object.book_set.all %}
```

Comment:
Please change the default name of `object` to make our code more readable .. it could be `category` ..